storage/spanstore/metrics/decorator_test.go

```diff
@@ -43,14 +43,14 @@ func TestSuccessfulUnderlyingCalls(t *testing.T) {
 	mrs.FindTraces(context.Background(), &spanstore.TraceQueryParameters{})
 	counters, gauges := mf.Snapshot()
 	expecteds := map[string]int64{
-		"get_operations|result=ok":  1,
-		"get_operations|result=err": 0,
-		"get_trace|result=ok":       1,
-		"get_trace|result=err":      0,
-		"find_traces|result=ok":     1,
-		"find_traces|result=err":    0,
-		"get_services|result=ok":    1,
-		"get_services|result=err":   0,
+		"get_operations.calls|result=ok":  1,
+		"get_operations.calls|result=err": 0,
+		"get_trace.calls|result=ok":       1,
+		"get_trace.calls|result=err":      0,
+		"find_traces.calls|result=ok":     1,
+		"find_traces.calls|result=err":    0,
+		"get_services.calls|result=ok":    1,
+		"get_services.calls|result=err":   0,
```

Comment:
what does the full name look like?  Perhaps the operation name should be pushed into a tag as well: `requests|operation=find_traces|result=ok`

Reply:
Sounds good - the updated version is now:
```
jaeger_query_latency_bucket{operation="find_traces",result="err",le="0.005"} 0
jaeger_query_latency_bucket{operation="find_traces",result="ok",le="0.005"} 2
jaeger_query_requests{operation="find_traces",result="err"} 0
jaeger_query_requests{operation="find_traces",result="ok"} 2
jaeger_query_responses_bucket{operation="find_traces",le="0.005"} 2
```